crypto/common/example_test.go

```diff
@@ -0,0 +1,86 @@
+package common_test
+
+import (
+	"fmt"
+
+	"go.dedis.ch/dela/crypto/bls"
```

Comment:
Kyber's BLS indicates that it's deprecated because it's got security flaws.
We probably shouldn't use it for the example as it may give some bad ideas :) 

May I suggest to replace it with bdn?

Reply:
Like we discuss, BDN fixes the use of rogue public key which is possible in certain circumstances and BDN is also drawbacks, so it's a matter of deciding. BLS itself works, so the example is not flawed.